docs/source/notebooks/metadata/metadata.txt

```diff
@@ -0,0 +1,464 @@
+    "feature_image": "./img/thz_ring_resonator.png"
+  },
+  "VizData.ipynb": {
+    "title": "Visualization of Simulation Data in Tidy3D | Flexcompute",
+    "description": "This notebook demonstrates how to perform visualization of simulation data in Tidy3D FDTD.",
+    "keywords": "result visualization, data plotting, Tidy3D, FDTD",
+    "feature_image": ""
+  },
+  "VizSimulation.ipynb": {
+    "title": "Visualization of Geometries in Tidy3D | Flexcompute",
+    "description": "This notebook demonstrates how to visualize geometries in Tidy3D FDTD.",
+    "keywords": "geometries, visualization, Tidy3D, FDTD",
+    "feature_image": ""
+  },
+  "WaveguideCrossing.ipynb": {
+    "title": "Waveguide Crossing Modeling in Tidy3D | Flexcompute",
+    "description": "This notebook demonstrates how to model a waveguide crossing in Tidy3D FDTD.",
+    "keywords": "waveguide crossing, Tidy3D, FDTD",
+    "feature_image": "./img/waveguide_crossing.png"
+  },
+  "WaveguidePluginDemonstration.ipynb": {
+    "title": "Using the Waveguide Plugin in Tidy3D | Flexcompute",
+    "description": "This notebook demonstrates how to use the waveguide plugin to analyze optical modes in photonic integrated devices in Tidy3D FDTD.",
+    "keywords": "waveguide, mode analysis, waveguide mode, waveguide plugin, Tidy3D, FDTD",
+    "feature_image": ""
+  },
+  "WaveguideSizeConverter.ipynb": {
+    "title": "Waveguide Size Converter Modeling in Tidy3D | Flexcompute",
+    "description": "This notebook demonstrates how to model waveguide size converters in Tidy3D FDTD.",
+    "keywords": "waveguide size converter, taper, luneburg lens, Tidy3D, FDTD",
+    "feature_image": "./img/luneburg_lens_taper.png"
+  },
+  "WaveguideToRingCoupling.ipynb": {
+    "title": "Modeling the Waveguide to Ring Coupling | Flexcompute",
+    "description": "This notebook demonstrates how to model a waveguide to ring coupling and avoid simulation divergence.",
+    "keywords": "waveguide, ring, coupling, absorber, diverge, silicon photonics, PIC, Tidy3D, FDTD",
+    "feature_image": "./img/waveguide_to_ring.png"
+  },
+  "WebAPI.ipynb": {
+    "title": "Running Simulations Through the Cloud | Flexcompute",
+    "description": "This notebook demonstrates how to run simulations through the cloud in Tidy3D FDTD.",
+    "keywords": "web API, job, batch, Tidy3D, FDTD",
+    "feature_image": ""
+  },
+  "YJunction.ipynb": {
+    "title": "Waveguide Y Junction Modeling in Tidy3D | Flexcompute",
+    "description": "This notebook demonstrates how to simulate a waveguide Y junction in Tidy3D FDTD.",
+    "keywords": "Y junction, T branch, Tidy3D, FDTD",
+    "feature_image": "./img/y_junction_schematic.png"
+  },
+  "ZeroCrossTalkTE.ipynb": {
+    "title": "Exceptional Coupling for Crosstalk Reduction | Flexcompute",
+    "description": "This notebook investigates the utilization of anisotropic metamaterial to reduce waveguide crosstalk by leveraging the exceptional coupling.",
```

Comment:
"an" anisotropic metamaterial

Reply:
"leveraging exceptional coupling" (I think?)

Reply:
Thanks for the great catches!